src/DataObjectAnnotator.php

```diff
@@ -52,13 +57,13 @@ class DataObjectAnnotator extends Object
 
     /**
      * DataObjectAnnotator constructor.
+     * @throws \Psr\Container\NotFoundExceptionInterface
```

Comment:
Prefer import over listing fully qualified

Reply:
Haven't gone through all of them yet.